### PR TITLE
[Try assistant modal] Hide fullscreen/picker buttons in inputbar

### PR DESCRIPTION
**Pull request by philipperolet**
## Description
Cf title
image

### CODE REVIEW

File: front/components/assistant/conversation/input_bar/InputBarContainer.tsx [hunk 1]
```diff
@@ -126,24 +128,28 @@ const InputBarContainer = ({
...
-          <div className="hidden sm:flex">
-            <IconButton
-              variant={"tertiary"}
-              icon={isExpanded ? FullscreenExitIcon : FullscreenIcon}
-              size="sm"
-              className="flex"
-              onClick={handleExpansionToggle}
-            />
-          </div>
+          {!isInTryModal && (
```
[the remark this answers is not in the document]
Comment (philipperolet) [reply front/components/assistant/conversation/input_bar/InputBarContainer.tsx]: Fair, changing :+1:

File: front/components/assistant/conversation/input_bar/InputBarContainer.tsx [hunk 2]
```diff
@@ -31,6 +31,7 @@ export interface InputBarContainerProps {
   owner: WorkspaceType;
   selectedAssistant: AgentMention | null;
   stickyMentions: AgentMention[] | undefined;
+  hideQuickActions: boolean;
```
Comment (flvndvd) [inline front/components/assistant/conversation/input_bar/InputBarContainer.tsx:34]: Let add some default props:

```
InputBarContainer.defaultProps = {
hideQuickActions: false
};
```

**Review by flvndvd — COMMENTED** [review]
LGTM, left a last comment.

File: front/components/assistant/conversation/input_bar/InputBarContainer.tsx (hunk 2 above)
Comment (philipperolet) [reply front/components/assistant/conversation/input_bar/InputBarContainer.tsx:34]: Not needed IMHO, there's only one call site, that actually uses the prop; I prefer to add code when it becomes needed if you don't mind (+ even if we reused it's debatable whether the prop should be implicitely set everywhere or not)

Comment (flvndvd) [reply front/components/assistant/conversation/input_bar/InputBarContainer.tsx:34]: Fine.

**Review by flvndvd — APPROVED** [review]

**Pull request merged**